docs/Runtime_Config.md

```diff
@@ -7,40 +7,41 @@ You can run it by providing the minimal configuration values as listed below. Pl
+| DTRACK_URL                              |                                               | Baseurl of Dependency-Track                                                                                        |
+| CDXGEN_USE_GOSUM                        | false                                         | see [cdxgen](https://github.com/CycloneDX/cdxgen#environment-variables)                                            |
+| CDXGEN_REQUIRED_SCOPE_ONLY_DEFAULT      | false                                         | Only include _required_ scope to created BOM (exclude test scope)                                                  |
+| CDXGEN_EVIDENCE_DEFAULT                 | false                                         | Create sbom with evidence (slows down the process)                                                                 |
+| CDXGEN_FORMULATION_DEFAULT              | false                                         | Generate formulation section using git metadata.                                                                   |
+| ANALYSIS_RECURSIVE_DEFAULT              | true                                          | default value for the `analysis.recursvie` config                                                                  |
+| APP_CLEAN_WRAPPER_SCRIPTS               | false                                         | Remove wrapper scripts like gradlew or mvnw for not downloading these tools                                        |
+| APP_ANALYSIS_TIMEOUT                    | 60M                                           | Maximal duration of an analysis before getting aborted                                                             |
+| APP_ENABLED_REPOS                       |                                               | Comma separated list of repo names that should be analyzed; all if empty                                           |
+| APP_PROCESS_LOGLEVEL                    | INFO                                          | Log config for OS commands like 'cdxgen', set to 'DEBUG' to see its output                                         |
+| SENSITIVE_ENV_VARS                      | see [Dockerfile](/src/main/docker/Dockerfile) | Comma separated list of env var names, that must not be logged                                                     |
+| ALLOWED_ENV_SUBSTITUTIONS               | see [Dockerfile](/src/main/docker/Dockerfile) | Comma separated list of env var names, that can be used in repo config                                             |
+| GRYPE_TEMPLATE                          | see [Dockerfile](/src/main/docker/Dockerfile) | Template to be used by grype for vulnerability reports in pull-requests                                            |
+| DEPSCAN_TEMPLATE                        | see [Dockerfile](/src/main/docker/Dockerfile) | Template to be used by depscan for vulnerability reports in pull-requests                                          |
```

Comment:
maybe this can be simplified to
```suggestion
| SENSITIVE_ENV_VARS                      | see [Dockerfile] | Comma separated list of env var names, that must not be logged                                                     |
| ALLOWED_ENV_SUBSTITUTIONS               | see [Dockerfile] | Comma separated list of env var names, that can be used in repo config                                             |
| GRYPE_TEMPLATE                          | see [Dockerfile] | Template to be used by grype for vulnerability reports in pull-requests                                            |
| DEPSCAN_TEMPLATE                        | see [Dockerfile] | Template to be used by depscan for vulnerability reports in pull-requests                                          |
```

and add this
```md
[Dockerfile]: /src/main/docker/Dockerfile
```

at the bottom of this table?

Reply:
I just fixed the already existing links to the dockerfile as the links were broken.